editoast/src/views/train_schedule/mod.rs

```diff
@@ -527,10 +529,13 @@ async fn create_backend_request_payload(
         })?;
 
     let electrical_profile_set = match scenario.electrical_profile_set_id {
-        Some(Some(id)) => {
+        Some(Some(electrical_profile_set_id)) => {
             let mut conn = db_pool.get()?;
-            let eps = ElectricalProfileSet::retrieve(&mut conn, id)?;
-            eps.name
+            ElectricalProfileSet::retrieve_conn(&mut conn, electrical_profile_set_id)?
```

Comment:
Likewise, `retrive(pool, id)` (there should have been a block wrapping the conn here 👀)